### PR TITLE
Optional upgrade step to run installer

### PR DESCRIPTION
**Pull request by ares**
I expect some discussion on mailing list first, then we'll update/merge/close this PR.

### CODE REVIEW

**Comment by lzap** [comment]
I like this idea, but I've been told that `--noop` in Puppet mean something that can break (while proper run works fine). :+1:

**Comment by ares** [comment]
@lzap yes, this can happen if you have two resources, A depends on updating B but in `--noop` B won't  update so A fails, I think during upgrades it should not happen too ofter

File: _includes/manuals/1.6/3.6_upgrade.md [hunk 1]
```diff
@@ -154,6 +154,21 @@ Debian users should reinstall plugins if desired using apt at this point.
 
 etc.
 
+##### Optional Step 3 (C) - Run foreman-installer
+
+If you used foreman-installer for existing foreman instance we recommend
```
Comment (domcleal) [inline _includes/manuals/1.6/3.6_upgrade.md]: s/for existing foreman instance/to set up your Foreman instance/

File: _includes/manuals/1.6/3.6_upgrade.md [hunk 2]
```diff
@@ -154,6 +154,21 @@ Debian users should reinstall plugins if desired using apt at this point.
...
+running it again after upgrading. Note that installer can modify config
+files so this may overwrite your custom changes. You can run installer
```
Comment (domcleal) [inline _includes/manuals/1.6/3.6_upgrade.md]: s/installer/the installer/ (above and below) too

**Comment by ares** [comment]
fixed

**Comment by domcleal** [comment]
Thanks @ares!

**Pull request merged**